.claude/context/submodule-workflow.md

```
@@ -0,0 +1,167 @@
+# PMOVES.AI Submodule Workflow
+
+This document describes the proper workflow for working with PMOVES.AI submodules.
+
+## Branch Strategy
+
+### Branch Flow
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)